app/controllers/admin/auction_mark_payments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Admin::AuctionMarkPaymentsController < Admin::BaseController
+  def update
+    @auction = Auction.find(params[:id])
+
+    paid_auction = MarkPaidAuction.new(
```

Comment:
ok cool, then let's just make sure we rename it so it is obvious that this is how we mark "Other P Card" auctions as paid, not all auctions